src/TheThingsNetwork.h

```diff
@@ -23,15 +20,18 @@
 
 #define TTN_BUFFER_SIZE 300
 
-// The Things Industries devices
+// The Things Products devices
 #if defined(ARDUINO_THINGS_NODE) || defined (ARDUINO_THINGS_UNO)
 typedef HardwareSerial SerialType;
+#define HARDWARE_UART
 
 #elif defined(ARDUINO_ARCH_SAM) || defined(ARDUINO_ARCH_SAMD)
 typedef HardwareSerial SerialType;
+#define HARDWARE_UART
 
 #elif defined(ARDUINO_ARCH_SAM) || defined(ARDUINO_ARCH_SAMD)
```

Comment:
Why are you checking exactly the same thing as on L28?

Reply:
Because it's a mistake, it's fixed